lumigator/python/mzai/sdk/README.md

```diff
@@ -3,16 +3,16 @@
 The SDK provides the communication and validation primitives needed to contact the Lumigator itself,
 either locally or remotely.
 
-You can install the lumigator SDK  via `pip` directly or via `uv`:
+You can install the lumigator SDK  via `pip` directly or via `uv` - currently after cloning the repository. Please note that you will need to install `schemas` as well:
```

Comment:
When you install the SDK, `schemas` is installed automatically because of [this](https://github.com/mozilla-ai/lumigator/blob/092dbca7be4eb34a8ea1d054c63dd3ee8ec966d9/lumigator/python/mzai/sdk/pyproject.toml#L19).

Reply:
Hm, you're right. Installing for me on a new venv had failed unless I installed it explicitly, but I cannot reproduce now, so out it goes.